_inc/client/plans/product-selector.jsx

```diff
@@ -184,12 +214,26 @@ class ProductSelector extends Component {
 			);
 		}
 
+		const isFetching = isFetchingData || ! plans;
```

Comment:
Minor: almost feels like `|| ! plans` should be part of the logic that computes `isFetchingData` in `mapStateToProps`.

Reply:
It makes a lot of sense! We don't really need `plans` inside the component - it's just used for signalling if the data is already fetched or not. I've removed it completely from the component, leaving just a constant in `connect` function (see c9cb90b).

Reply:
Awesome; 🚢  it!